project_stages.md

```diff
@@ -0,0 +1,176 @@
+* Projects must have documented, initial project governance.
+
+#### Project Process: Sandbox to Incubating
+
+Incubating Projects are required to undergo due diligence as a part of the process to move from Sandbox to Incubation. Due Diligence is driven by a TAC or parent WG sponsor. Once the diligence is complete and the proposal made, two weeks are allocated for public comment before a TAC vote is called.
+
+
+#### Submission Forms for Sandbox to Incubating
+
+1. [Proposal](TBD)
+2. [Legal](TBD)
+
+
+### Graduated
+
+Graduated projects signal the highest level of maturity for an OpenSSF project. Graduated projects have consistent, mature open source development practices with sustained contribution and activity. 
+
+#### Project Responsibilities
+
+* Bi-annual updates to the TAC on technical vision and progress on vision
+* Maintain a diversified contributor base (i.e. not a single-vendor project) with an active flow of contributions
+* Follow security best practices (as recommended by the OpenSSF and others)
+* Maintain a point of contact for vulnerability reports and follow coordinated vulnerability disclosure practices
+* Implement, practice, and refine mature software development and release practices such a following a version schema
+
+#### Project Support
+* Receives guidance on technical direction from TAC 
+* Receives space for Contributor Summits at OpenSSF bi-annual events
+* Receives OpenSSF Code of Conduct Committee support
+* Receives infrastructure support (details determined by project leads and OpenSSF Budget Committee)
+* Receives support with vulnerability disclosure from the OpenSSF (Vulnerability Disclosure WG)
+* May post project updates and tutorials to the OpenSSF blog
+* May request OpenSSF budget for project improvements such as security audits or time-bound contracting needs
+* May request OpenSSF for sustained maintainer stipends (details determined by OpenSSF and project leads)
+* With additional TAC or WG approval, may fundraise for dedicated project funds, coordinated by the OpenSSF
+* Projects may use the OpenSSF logo to promote their project (in accordance with the trademark guidelines). Projects may be referred to as an "OpenSSF Project" or "OpenSSF $ProjectName." 
+
+#### Graduated Project Entry Requirements and Considerations
+
+* Projects must have a minimum of three maintainers with a minimum of two different company affiliations.
+* Projects must be aligned with the OpenSSF mission and be a novel approach for existing areas or address an unfulfilled need. Projects with duplicate, similar or competing fuctionality to an existing OpenSSF project may be denied Graduation status if the TAC does not see technical justification for overlapping projects. 
+* Projects must be able to show adoption by multiple parties and that adoption's value to the open source community and or end users.
```

Comment:
Re "adoption" (vs production usage) and "multiple parties" (vs companies), one of the use cases I was thinking of was an open source project using an OpenSSF project in their build and release workflow.

Reply:
That makes a lot of sense. Since this should be a measure of maturity, and I agree that adoption in certain open source projects would demonstrate that, how could this be worded to avoid gaming this metric by demonstrating adoption in very small / low impact projects?

Reply:
That is a great question! 

"critical adoption" by "multiple, significant parties"?

Reply:
vague and open-ended? I like it ... and at the same time, I think the context you so clearly articulated on the TAC call would be lost.

Perhaps a second clarifying sentence, or a parenthetical, could help? e.g., 

```
* Projects must be able to show adoption by multiple parties, which could be production deployments or substantial use by established external open source communities, and demonstrate the value of that adoption to either the end users or the open source community.
```